taipy/gui/builder/_element.py

```diff
@@ -67,6 +66,12 @@ def update(self, **kwargs):
         self._properties.update(kwargs)
         self.parse_properties()
 
+    @staticmethod
+    def __get_lambda_index():
+        _Element.__LAMBDA_VALUE_IDX += 1
+        _Element.__LAMBDA_VALUE_IDX %= 10000
```

Comment:
Is this necessary?
If it's a fear for overflow, why not a far larger value? closer to MAX_INT?

Reply:
I know ...
what do you think ?